### PR TITLE
Add fps output, used by NFR pipeline

### PR DESCRIPTION
**Pull request by m-samiec**
FPS output is required by NFR pipeline to deduce the health of workers.

### CODE REVIEW

File: LaunchSimClient.bat [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@echo off
```
Comment (m-samiec) [inline LaunchSimClient.bat]: This is a driveby utility script

File: Game/Source/GDKTestGyms/GDKTestGymsGameInstance.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+public:
+	virtual void Init() override;
+	bool Tick(float DeltaSeconds);
+
+private:
+	FTickerDelegate TickDelegate;
+	FDelegateHandle TickDelegateHandle;
+
+	float AverageFPS = 1.0f;
```
Comment (improbable-danny) [inline Game/Source/GDKTestGyms/GDKTestGymsGameInstance.h]: Is 30/60 a better starting point?

Comment (improbable-danny) [reply Game/Source/GDKTestGyms/GDKTestGymsGameInstance.h]: Maybe higher is better if were going to fail tests on < x fps, the bias is removed after 30 frames from the code above
